ballerina/client.bal

```diff
@@ -24,7 +24,7 @@ public isolated client class Client {
     # + config - The configurations to be used when initializing the `connector` 
     # + serviceUrl - URL of the target service 
     # + return - An error if connector initialization failed 
-    public isolated function init(ConnectionConfig config =  {}, string serviceUrl = "https://demo.docusign.net/clickapi") returns error? {
+    public isolated function init(string serviceUrl, ConnectionConfig config =  {}) returns error? {
```

Comment:
May I know why this serverURL is made as the required parameter?

Reply:
There are two endpoints can be used in DocuSign integration: one for the developer environment and another for the production environment. https://developers.docusign.com/docs/esign-rest-api/go-live/
Therefore the parameter is provided as required, allowing developers to explicitly configure which endpoint they want to use.